src/Livewire/DataTables/MediaList.php

```diff
@@ -2,15 +2,27 @@
 
 namespace FluxErp\Livewire\DataTables;
 
+use FluxErp\Actions\Media\DeleteMedia;
+use FluxErp\Livewire\Forms\MediaForm;
 use FluxErp\Models\Media;
 use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Validation\ValidationException;
+use Livewire\Attributes\Renderless;
+use Spatie\Permission\Exceptions\UnauthorizedException;
 use Symfony\Component\HttpFoundation\BinaryFileResponse;
 use TeamNiftyGmbH\DataTable\Htmlables\DataTableButton;
+use WireUi\Traits\Actions;
 
 class MediaList extends BaseDataTable
 {
+    use Actions;
+
     protected string $model = Media::class;
 
+    public ?string $includeBefore = 'flux::livewire.datatables.media.media-list';
```

Comment:
why not match view name with qualified class name? 'flux::livewire.datatables.media-list'?